### PR TITLE
Don't append a slash to file paths

### PR DESCRIPTION
**Pull request by jakemcdermott**
for #9701

### CODE REVIEW

File: awx/ui_next/src/screens/Inventory/shared/InventorySourceSubForms/SCMSubForm.jsx [hunk 1]
```diff
@@ -122,20 +122,13 @@ const SCMSubForm = ({ autoPopulateProject, i18n }) => {
...
           }}
           aria-label={i18n._(t`Select source path`)}
           placeholder={i18n._(t`Select source path`)}
           isCreatable
           onCreateOption={value => {
             value.trim();
-
-            if (!value.endsWith('/')) {
-              value += '/';
-            }
```
Comment (jakemcdermott) [inline awx/ui_next/src/screens/Inventory/shared/InventorySourceSubForms/SCMSubForm.jsx:138]: This _may_ be addressing a different use case or bug that I am not aware of...

Comment (jakemcdermott) [reply awx/ui_next/src/screens/Inventory/shared/InventorySourceSubForms/SCMSubForm.jsx:138]: @AlexSCorey Can these custom inventories ever be whole directories? :thinking: (Trying to make sure I don't miss something by removing this)

**Review by shanemcd — APPROVED** [review]

File: awx/ui_next/src/screens/Inventory/shared/InventorySourceSubForms/SCMSubForm.jsx (hunk 1 above)
Comment (jakemcdermott) [reply awx/ui_next/src/screens/Inventory/shared/InventorySourceSubForms/SCMSubForm.jsx:138]: I feel like removing this code is _probably_ fine? Although I can't think of a use case off the top of my head, if someone does need to append a `/` they can still do it manually:

inv_slash

cc @AlexSCorey - thoughts?

**Review by AlexSCorey — APPROVED** [review]
LGTM

File: awx/ui_next/src/screens/Inventory/shared/InventorySourceSubForms/SCMSubForm.jsx [hunk 2]
```diff
@@ -126,6 +126,7 @@ const SCMSubForm = ({ autoPopulateProject, i18n }) => {
           }}
           aria-label={i18n._(t`Select source path`)}
           placeholder={i18n._(t`Select source path`)}
+          createText={i18n._(t`Set source path to`)}
```
Comment (jakemcdermott) [inline awx/ui_next/src/screens/Inventory/shared/InventorySourceSubForms/SCMSubForm.jsx:129]: The default text shown when manually setting an inventory source is kind of weird:

Screenshot from 2021-03-25 18-32-23

"Create foo/bar" makes it seem like a file or directory would be created somewhere when it wouldn't be. 

---

When we type in a path like this, we're really just specifying a file location that may or may not exist. Our component library has a [parameter for customizing the text](https://www.patternfly.org/2020.04/documentation/react/components/select) so I changed it:

Screenshot from 2021-03-25 18-30-54

Comment (AlexSCorey) [reply awx/ui_next/src/screens/Inventory/shared/InventorySourceSubForms/SCMSubForm.jsx:129]: I like this phrasing

**Comment by jakemcdermott** [comment]
note: [branch results](https://dashboard.cypress.io/projects/ui2mq7/runs/5081/overview) currently match [devel results](https://dashboard.cypress.io/projects/ui2mq7/runs/5077/overview)

**Pull request merged**